### ISSUE TITLE
Functionality Addon

### ISSUE THREAD

**Issue by farazhaider88** [quran/quran-ios#73]
assalam o alakum,

there is an ios app -> https://itunes.apple.com/us/app/quran-memorizer/id421890227?mt=8

Can we mimic some of the functionalities in our quran application. it contain some much needed functionality which our application lacks. like Page Flip animation, Translation, looping audio, e.t.c

**Comment by mohamede1945** [comment]
We alikom elsalam,

Thank you so much for your request. Currently, we have Translation, Audio looping in our roadmap. Translation is our top priority right now. 

If you have time and would like to contribute in any feature that would be awesome.

Jazaka Allah Khyaran

**Comment by farazhaider88** [comment]
sure it will be honour to contribute in such a beneficial cause. i would take out some time and work on this.

one more thing I did not find any documentation and tutorial regarding each version. i think including this will encourage more people to contribute on this holy cause.

**Comment by mohamede1945** [comment]
You mean something like this https://github.com/quran/quran-ios/releases ?

**Comment by mohamede1945** [comment]
We will consider Page Flip Animation, other issues reported are duplicate of #4 #35

**Comment by abunur** [comment]
I strongly suggest not incorporating page-flip animation.  The app is currently clean and simple.  Page flip does not contribute to the user experience in any meaningful way.

**Comment by mohamede1945** [comment]
I definitely see your point. But I wouldn't say totally no to it, as it can be a switch similar to iBooks do it. What do you think? Also, it is low priority one.

**Comment by ahmedre** [comment]
just my $0.02 - we used to have it in Quran for Android and removed it. i like the cleaner / simpler / faster page flips than with the animation, but that's just my humble opinion.

**Comment by mohamede1945** [comment]
But you had it as a setting or as it is the only way to change pages? Because I still see people get amused for good animation. Personally, I would still use scrolling.

**Comment by ahmedre** [comment]
no, we had it as the only way back then.

**Comment by mohamede1945** [comment]
I'm still with you guys that scrolling is very very important but if we have time we can implement page flip animation as a secondary way to change pages while keeping scrolling as default.

**Comment by abunur** [comment]
Personally I think the only reason it's still in iBooks is Apple has neglected the app.  Everything else in general they have shifted away from the paradigm of imitating real world objects in favor of a clean and simple interface

**Comment by mohamede1945** [comment]
I totally agree. But I still see people (including myself 😃) get amused of similar animations and I usually try to do it multiple times.
